selftests/lint.sh

```diff
@@ -9,4 +9,4 @@ MODULES="avocado contrib examples scripts docs optional_plugins selftests"
 ${PYLINT} ${PYLINT_OPTIONS} --disable=W0212,W0511,W0703,W0707,R,C,E1101,E1120,E0401,I0011 --enable=R0205,R0401,W1601,W1602,W1603,W1604,W1605,W1606,W1607,W1608,W1609,W1610,W1611,W1612,W1613,W1614,W1615,W1616,W1617,W1620,W1621,W1622,W1623,W1624,W1625,W1626,W1627,W1628,W1629,W1630,W1634,W1635,W1636,W1637,W1638,W1639,W1640,W1642,W1643,W1644,W1645,W1646,W1647,W1648,W1649,W1650,W1651,W1652,W1653,W1654,W1655,W1656,W1657,W1658,W1659,W1660,W1661,W1662 ${MODULES}
 
 MODULES="avocado contrib examples scripts docs optional_plugins"
-${PYLINT} ${PYLINT_OPTIONS} --enable=W0212,W0703 ${MODULES}
+${PYLINT} ${PYLINT_OPTIONS} --enable=W0212,W0703,R0201 ${MODULES}
```

Comment:
@clebergnu the only conflict that I can see with #4516 is that R0201 will be enabled by default with `--enable=all`. So this will be unnecessary.